pkg/restore/crd.go

```diff
@@ -94,38 +101,75 @@ func RestoreCrd(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorN
 		logging.Fatal(fmt.Errorf("failed to delete validating webhook %s. %v", webhookConfigName, err))
 	}
 
-	removeOwnerRefOfUID(ctx, k8sclientset, operatorNamespace, clusterNamespace, string(crdResource.GetUID()))
+	removeOwnerRefOfUID(ctx, k8sclientset, operatorNamespace, clusterNamespace, string(crdResource.GetUID()), customResources)
 
 	logging.Info("Removing finalizers from %s/%s", crd, crName)
 
 	jsonPatchData, _ := json.Marshal(crds.DefaultResourceRemoveFinalizers)
-	err = k8sclientset.PatchResourcesDynamically(ctx, crds.CephRookIoGroup, crds.CephRookResourcesVersion, crd, clusterNamespace, crName, types.MergePatchType, jsonPatchData)
+	err = k8sclientset.PatchResourcesDynamically(ctx, groupName, versionResource, crd, clusterNamespace, crName, types.MergePatchType, jsonPatchData)
 	if err != nil {
-		logging.Fatal(fmt.Errorf("Failed to update resource %q for crd. %v", crName, err))
+		logging.Fatal(fmt.Errorf("Failed to update resource %q for crd.  %v", crName, err))
```

Comment:
nit: no need for extra space
```suggestion
		logging.Fatal(fmt.Errorf("Failed to update resource %q for crd. %v", crName, err))
```

Reply:
done.